clio-server/src/main/scala/org/broadinstitute/clio/server/dataaccess/SearchDAO.scala

```diff
@@ -33,18 +29,6 @@ trait SearchDAO {
     */
   def close(): Future[Unit]
 
-  /**
```

Comment:
Did you mean to delete this javadoc? It looks like it could apply to the method that still remains.

Reply:
I did not.